lib/blueflood_http_client.js

```diff
@@ -19,6 +19,7 @@
 var url = require('url');
 var http = require('http');
 var request = require('request');
+var Logger = require('log4js').getLogger();
```

Comment:
IMHO NodeJS coding conventions are inconsistent when it comes to module imports, but in this case to be consistent with the others above it, this should probably be lowercase `logger`.